l10n_de_tax_statement/models/l10n_de_tax_statement_config.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 BIG-Consulting GmbH(<http://www.openbig.org>)
+# Copyright 2019 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class VatStatementConfig(models.Model):
```

Comment:
Modelname differs a little bit from class name sometimes....

Reply:
In the [OCA Guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst) is not mentioned that the class name should be the same as the model name.

Changing the class name in UpperCamelCase would end up in "_L10nDeTaxStatementConfig_". Not so readable IMO. Is it ok to keep the class name as it is?